src/app/(post)/search/_components/SearchBar.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+
+import Icon from '@/src/components/Icon';
+import TextInput from '@/src/components/TextInput';
+
+interface SearchBarProps {
+  onClear: () => void;
+  onSubmit: (input: string) => void;
+}
+
+const SearchBar: React.FC<SearchBarProps> = ({ onClear, onSubmit }) => {
+  const [input, setInput] = useState('');
+
+  const handleInputChange = (e: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
```suggestion
  const handleInputChange = (e: React.ChangeEvent<HTMLInputElement>) => {
```

여기도 ChangeEvent를 모듈로 불러와서 `React.`을 제거해주면 통일성이 생길 것 같습니다.